portfolio/src/main/webapp/journal.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <title>Journal</title>
+  <meta name="description" content="The HTML5 Herald">
+  <meta name="author" content="SitePoint">
+
+  <link rel="stylesheet" href="css/styles.css?v=1.0">
+
+</head>
+
+<body>
+  <script src="journal.js"></script>
+  <form action="/journal" method="POST">
+
+        <p id = "date" > July 1st, 2020 </p>
+
+        <!-- journal editing body
+        todo: text box needs to cover page and be transparent -->
+
+        <input type ="text" name = "journal-input">
```

Comment:
This should probably be a textarea.  Writing journal entries into a text input would be a bit of a pain

Reply:
Thanks for the suggestion, the layout looks much better. fixed.